examples/drivenets/cdnos.pb.txt

```diff
@@ -0,0 +1,29 @@
+name: "cdnos-demo"
+nodes: {
+    name: "cdnos1"
+    vendor: DRIVENETS
+    config: {
+        config_file: "default"
```

Comment:
config_file and config_path should be defaulted to simplify topo (if these are standard values), see other vendors node impls

Reply:
added default config file and path for node